apps/desktop/src/main.ts

```diff
@@ -262,7 +263,7 @@ export class Main {
             this.nativeMessagingMain.generateDdgManifests().catch(this.logService.error);
           }
 
-          this.nativeMessagingMain.listen();
+          this.nativeMessagingMain.listen().catch(this.logService.error);
```

Comment:
```suggestion
          this.nativeMessagingMain.listen().catch((err) => this.logService.error("Error while starting native message listener", err));
```

I like to add a little contextual message that is searchable because JS stack traces can be a bit undecipherable sometimes (most of the time).

(Probably can't accept this suggestion as is because it probably wouldn't make lint happy)

Reply:
Good point, the error messages are much clearer that way